cluster/gce/config-test.sh

```diff
@@ -49,7 +49,7 @@ CLUSTER_NAME="${CLUSTER_NAME:-${INSTANCE_PREFIX}}"
 MASTER_NAME="${INSTANCE_PREFIX}-master"
 MASTER_TAG="${INSTANCE_PREFIX}-master"
 NODE_TAG="${INSTANCE_PREFIX}-minion"
-CLUSTER_IP_RANGE="${CLUSTER_IP_RANGE:-10.245.0.0/16}"
+CLUSTER_IP_RANGE="${CLUSTER_IP_RANGE:-10.245.0.0/14}"
```

Comment:
In theory someone can run tests on the same GCE network as their main cluster. (They have to override the `NETWORK`, obviously.)

At the very least, the code should use the canonical address of `10.244.0.0./14`, but I tend to agree with @roberthbailey